WordPress/Classes/ViewRelated/Domains/Domain registration/RegisterDomainSuggestions/DomainSuggestionViewControllerWrapper.swift

```diff
@@ -0,0 +1,38 @@
+import SwiftUI
+import UIKit
+
+/// Makes RegisterDomainSuggestionsViewController available to SwiftUI
+final class DomainSuggestionViewControllerWrapper: UIViewControllerRepresentable {
+
+    let blog: Blog
+
+    weak var presentingController: RegisterDomainSuggestionsViewController?
```

Comment:
These could be private.

Reply:
Additionally, I think `presentingViewController` should probably be `presentedViewController`.

I'm using this as my reference for the naming suggestion: https://developer.apple.com/documentation/uikit/uiviewcontroller/1621430-presentingviewcontroller

Reply:
👍